apps/rush-lib/src/api/ChangeFile.ts

```diff
@@ -62,13 +62,15 @@ export class ChangeFile {
   }
 
   /**
-   * Writes the change file to disk in sync mode
+   * Writes the change file to disk in sync mode.
+   * Returns the filepath.
    */
-  public writeSync(): void {
+  public writeSync(): string {
```

Comment:
>string [](start = 22, length = 6)

Would it be better to return something like this?

```
interface IWriteSyncResult {
  filePath: string;
}
```

Reply:
not a public api so we can change later

---
In reply to: [210147349](https://github.com/Microsoft/web-build-tools/pull/769#discussion_r210147349) [](ancestors = 210147349)